### ISSUE TITLE
PDF (PDFAnno) Editor not shows up in Settings

### ISSUE THREAD

**Issue by JackDapid** [inception-project/inception#1505]
In the 'Settings -> General Display Preferences -> Editor' only the editors 'brat (sentence-oriented)' and 'brat (line-oriented)' oriented show up, but not 'PDF' (PDFAnno)

1. Go to 'Settings' icon while '...'
2. Unfold 'Editor' selection list
3. See only 'brat' editors

**Expected behavior**
It should allow to select PDF editor

**Screenshots**
<img width="590" alt="Screen Shot 2019-11-11 at 00 28 13" src="https://user-images.githubusercontent.com/8346230/68552792-5b438080-041b-11ea-9780-9aa937f6cfad.png">

**Please complete the following information:**
 - Version and build ID: INCEpTION -- 0.12.4 (2019-11-01 13:20:13, build 90c3d37ad69dee2b09a178f6748b47bede4264fa
 - OS: OS X
 - Browser: chrome

**Comment by reckart** [comment]
Add this line to your `settings.properties` file and restart, then the PDF editor should appear. Mind that you can only use the PDF view for documents imported with the "PDF format" in the project settings - a format which probably also only becomes available after adding this line:

```
ui.pdf.enabled=true
```

**Comment by JackDapid** [comment]
Thanks a lot, that did the job. The pdf document I had imported before (as PDF) worked fine, so the format was available before. I explicitly searched in the docs for a setting like this but didn't found any? Maybe it is there, maybe good to add to the documentation explicitly? As it is written in Admin doc that PDFAnno is a module, I found the separate inception-pdf-editor.jar and added it in the home folder (with same right)? Thanks a lot a really great experience so far. :-)  

`-rwxr-xr-x. 1 tomcat tomcat   3186727 Nov  1 13:19 inception-pdf-editor.jar`
`-rwxr-xr-x. 1 tomcat tomcat 228843109 Nov  1 13:27 inception.jar`
